EasyFinance.Domain/AccessControl/User.cs

```diff
@@ -26,6 +27,21 @@ public User(string firstName = "Default", string lastName = "Default", string pr
         public bool Enabled { get; set; } = true;
         public bool HasIncompletedInformation => string.IsNullOrEmpty(this.FirstName) && string.IsNullOrEmpty(this.LastName);
 
+        public void SetEmail(string email)
+        {
+            if (string.IsNullOrWhiteSpace(email))
+                throw new ValidationException(
+                    property: nameof(Email), 
+                    message: string.Format(ValidationMessages.PropertyCantBeNullOrEmpty, nameof(Email)));
+
+            if (!IsValidEmail(email)) 
+                throw new ValidationException(
+                    property: nameof(Email),
+                    message: string.Format(ValidationMessages.EmailFormatIsNotValid, nameof(Email)));
+
+            this.Email = email;
+        }
+
```

Comment:
```suggestion
```
Email is controlled by .Net Core 8 Authentication.

To update email you can use the UserManager. But I don't think that will be necessary in this scenario.